auth-api/src/auth_api/services/entity_mapping.py

```diff
@@ -209,12 +219,27 @@ def paginate_from_affiliations(org_id: int, search_details: AffiliationSearchDet
     def populate_affiliation_base(org_id: int, search_details: AffiliationSearchDetails):
         """Get entity details from the database and expand multiple identifiers into separate rows."""
         data = EntityMappingService.paginate_from_affiliations(org_id, search_details)
-        return [
+
+        affiliation_bases = [
             AffiliationBase(identifier=identifier, created=created)
             for identifiers, created in data
             for identifier in identifiers
         ]
 
+        if current_app.config.get("AFFILIATION_DEBUG") is True:
+            t_identifiers = [f'"{base.identifier}"' for base in affiliation_bases if base.identifier.startswith("T")]
+            nr_identifiers = [f'"{base.identifier}"' for base in affiliation_bases if base.identifier.startswith("NR")]
+            other_identifiers = [
+                f'"{base.identifier}"'
+                for base in affiliation_bases
+                if not base.identifier.startswith("T") and not base.identifier.startswith("NR")
+            ]
+            print(f"T identifiers ({len(t_identifiers)}): {', '.join(t_identifiers)}")
+            print(f"NR identifiers ({len(nr_identifiers)}): {', '.join(nr_identifiers)}")
+            print(f"Other identifiers ({len(other_identifiers)}): {', '.join(other_identifiers)}")
```

Comment:
These make it real easy to copy and paste from VSCODE -> Postman

Reply:
As discussed with Ody, we'll put in a simple logger for this one instead of using prints, but we need the logging.conf to come back